phpstan.neon.dist

```diff
@@ -54,6 +54,18 @@ parameters:
 				- tests/Fixtures/TestBundle/Document/
 				- tests/Fixtures/TestBundle/Entity/
 				- src/OpenApi/Factory/OpenApiFactory.php
+		-
+			message: '#is never assigned .* so it can be removed from the property type.#'
```

Comment:
false positive because `private ?int $id` is not set with a setter but with doctrine magic.